extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java

```diff
@@ -256,8 +256,13 @@ public FormAuthenticationMechanism get() {
                 String loginPage = form.loginPage.startsWith("/") ? form.loginPage : "/" + form.loginPage;
                 String errorPage = form.errorPage.startsWith("/") ? form.errorPage : "/" + form.errorPage;
                 String landingPage = form.landingPage.startsWith("/") ? form.landingPage : "/" + form.landingPage;
+                String postLocation = form.postLocation;
```

Comment:
To me it would make sense to auto-add `/` like for the `*Page` variables.

Reply:
Yes absolutely, I fixed that in my last commit